inst/tutorials/vaccines/vaccines.Rmd

```diff
@@ -0,0 +1,332 @@
+    individual_mosquitoes = FALSE
+  )
+)
+
+simparams <- set_equilibrium(parameters = simparams, init_EIR = starting_EIR)
+
+# Run a model with no interventions in a setting with no seasonality
+output_control <- run_simulation(timesteps = sim_length * 2, parameters = simparams)
+
+
+```
+
+
+## Mass vaccination strategy
+
+We can implement a mass vaccination strategy in which all people between 5 months and 50 years old are vaccinated with 3 doses of RTS,S, followed by a booster 12 months after:
+
+```{r mass_vaccination, exercise = TRUE}
+
+rtssmassparams <- set_mass_pev(
+  simparams,
+  profile = malariasimulation::rtss_profile, # We will model implementation of the RTSS vaccine.
+  timesteps = 1 * year,   # The single round of vaccination is at 1 year into the simulation.
+  coverages = 1,          # The vaccine is given to 100% of the population between the specified ages.
+  min_wait = 0,           # The minimum acceptable time since the last vaccination is 0 because in our case we are only implementing one round of vaccination.
+  min_ages = 5 * month,   # The minimum age for the target population to be vaccinated. 
+  max_ages = 50 * year,   # The maximum age for the target population to be vaccinated.
+  booster_timestep = 12 * month, # The booster is given at 12 months after the primary series. 
+  booster_coverage = 0.95,# Coverage of the booster dose is 95%.
+  booster_profile = list(malariasimulation::rtss_booster_profile) # We will model implementation of the RTSS booster.
+)
+
+sim_length <- 3 * year
+output <- run_simulation(timesteps = sim_length, parameters = rtssmassparams)
+
+# visualize outputs
+plot_inci(output)
+plot_prev(output)
+plot_doses(output)
+
+```
+
+```{r quiz2, echo= FALSE}
+quiz(caption = "Mass Vaccination Questions",
+          question("What modelled PFPR does the vaccine model achieve 2.5 years into the simulation?",
+              answer("0.40", correct =TRUE),
+              answer("0.24"),
+              answer("0.32"),
+              answer("0.21"), correct = "Correct-- the vaccine scenario achieves a parasite prevalence noticeably lower than the scenario where no vaccines are administered.", allow_retry = T),
+     question("About how many booster doses are administered in the vaccine simulation?",
+              answer("4000"),
+              answer("2000"),
+              answer("7000"),
+              answer("8000", correct = TRUE), correct = "Correct-- 8000 boosters are administered to the target population exactly 12 months after the third dose.", allow_retry = T)
+)
+
+```
+
+## Implement a mass vaccination strategy
+
+Use this space to implement a mass vaccination strategy that:
+
+- Is targeted at children under 5 years old
+- Includes a booster delivered 5 months after the first three doses
+- Reaches a coverage level of 50% for the first three doses and 75% for the booster dose.
+
+Plot incidence and the doses delivered for this model.
+
+```{r space_to_work, exercise = TRUE}
```

Comment:
Might be useful to add a hint/answer here
https://pkgs.rstudio.com/learnr/articles/exercises.html#r-code-hints